### ISSUE TITLE
Fullscreen video mode no longer working

### ISSUE THREAD

**Issue by traviswimer** [traviswimer/PlugPro#2]
At some point Plug.dj updated and the video mode is no longer working correctly. This probably just requires some styling changes.

**Comment by YourMomYourMomYourMomma** [comment]
My life was amazing once. I had a job that actually made monday mornings an enjoyable event. I had a beautiful wife—a jewish 'spinner' who did it all. I had season tickets to all my local professional sporting teams. Maybe I really was living a "dream," because one day I awoke to a nightmare... I signed into Plug, only to discover that my full-screen plug pro option was broken. Stricken with panic, I did not know what to do. When I realized I'd no longer be able to watch my favorite videos in full screen, I fell into depression and despair. Look many others in my situation, I turned to heroin and tranny hookers. My wife, disgusted with who I'd become, left me. Trying to fill the void at work, I turned to porn, ovbiosuly resulting in my termination. Even my dog abandoned me for a new owner. My life was amazing once. If only there was something someone could do...
